src/components/playerController/index.js

```diff
@@ -267,6 +277,50 @@ module.exports = class PlayerController {
     }
 
 
+    //================================================================
+    /**
+     * Processes automatic whitelist cleanup.
+     * Feature: Automatic revocation of whitelist if inactive
+     * @returns {void}
+     */
+    async processAutomaticWhitelistCleanup(){
+        if (!this.config.automaticWhitelistCleanup) {
+            return;
+        }
+
+        try {
+            const revokeDays = this.config.automaticWhitelistCleanupDays ? this.config.automaticWhitelistCleanupDays : 30;
+            const revokeTime = now() - (86400 * +revokeDays);
+            await this.dbo.get('players')
+                .filter(player => player.tsLastConnection <= revokeTime)
+                .value()
+                .forEach(async player => {
+                    let idArray = [];
```

Comment:
Can be `const`, you don't reassign its value.
```suggestion
                    const idArray = [];
```